src/index.ts

```diff
@@ -13,11 +13,9 @@ import { Connection } from '@hathor/wallet-lib';
 import logger from './logger';
 
 // @ts-ignore
-const machine = interpret(SyncMachine).start();
-
-machine.onTransition(state => {
-  if (state.changed && state.value) {
-    logger.debug(`Transitioned to state: ${state.value}`);
+const machine = interpret(SyncMachine).onTransition(state => {
```

Comment:
Merge conflict: I kept my decoupling of the `machine.start()` method. This can be useful later if we have the creating of the instances (machines, handlers, etc.) separated from the init of the service.